libraries/botframework-connector/src/auth/botFrameworkClientImpl.ts

```diff
@@ -36,6 +44,15 @@ export class BotFrameworkClientImpl implements BotFrameworkClient {
         ok(typeof botFrameworkClientFetch === 'function');
     }
 
+    /**
+     * @param fromBotId The MicrosoftAppId of the bot sending the activity.
+     * @param toBotId The MicrosoftAppId of the bot receiving the activity.
+     * @param toUrl The URL of the bot receiving the activity.
+     * @param serviceUrl The callback Url for the skill host.
+     * @param conversationId A conversation ID to use for the conversation with the skill.
+     * @param activity The Activity to send to forward.
+     * @returns Promise with optional invokeResponse.
```

Comment:
You could use the one from `botbuilder\src\botFrameworkHttpClient.ts`
`@returns {Promise<InvokeResponse<T>>} A promise representing the asynchronous operation.`